### ISSUE TITLE
The issue about the MRR metric after pre-training Graphcodebert differs from the MRR metric in the CoCoSoDa paper and GraphCodeBERT paper.

### ISSUE THREAD

**Issue by Zxb-QHU** [DeepSoftwareAnalytics/CoCoSoDa#6]
For example,
in your paper

GraphCodeBERT  for ruby.
MRR  is 0.703

However, under the premise of not changing settings, the MRR value is 0.702 after I fine-tuned it. Why is this the case?

Additionally, in CoCoSoda, the batch size was set to 128. When I increase the batch size or use multiple GPUs, the MRR value increases. However, your experimental results also show the same 0.703. Why is this?
